modules/auxiliary/dos/http/slow_loris.rb

```diff
@@ -0,0 +1,72 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Dos
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'            => 'Slow Loris DoS',
+      'Description'     => %q{Slowloris tries to keep many connections to the target web server open and hold them open as long as possible. 
```

Comment:
A few lines in the module have had spaces at EOL added back. Editor settings can sometimes help with that issue.